### ISSUE TITLE
[Not Exactly a Bug] Certain Amps Will Not Train

### ISSUE THREAD

**Issue by Simackomen** [sdatkinson/neural-amp-modeler#243]
Hi!
I ran into an issue with the Easy Mode Trainer as I was training my Orange Micro Dark model. The last best epoch (epoch 74) had the Error-to-Signal Ratio of 0.9865, and after 400 more epochs, the ESR stayed the same, which means that the model will be highly inaccurate. Here is a screenshot:
<img width="1440" alt="Screenshot 2023-05-13 at 5 35 15 PM" src="https://github.com/sdatkinson/neural-amp-modeler/assets/130309626/8ccb59c0-268f-40fe-b1a2-e65e4566ef43">

I suspect that the problem has to do with the learning rate of the neural software. For some amps, it only takes about 5 epochs for ESR to go down to 0.02, and for others, the ESR stays at about 0.20 regardless of the number of epochs. The Micro Dark I am currently training is an extreme case, of course, though there may be other amps that have similar signal curves and may be difficult to train. I think it would be a great idea if those (like me) who can't, for one reason or another, run NAM locally, to be able to tweak the learning rate in the Easy Mode Trainer besides the number of epochs.

**Comment by sdatkinson** [comment]
[It's already supported.](https://github.com/sdatkinson/neural-amp-modeler/blob/main/nam/train/colab.py#L69)

Provide e.g. `lr=0.001`.
